.NET agent: clarify that SQL query parameter capture is available, but not enabled by default (#21356)
newrelic/docs-website: src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements.mdx

            </tbody>
            </table>

>             The .NET agent doesn't directly monitor datastore processes. Also, by default the .NET SQL parameter capture in a query trace doesn't list parameters for a parameterized query or a stored procedure. Collection of the SQL query parameters can be enabled in the agent [configuration](/docs/apm/agents/net-agent/configuration/net-agent-configuration/).

Reviewer: ```suggestion
            By default, the .NET agent doesn't capture SQL parameters for stored procedures or parameterized queries in a query trace. The agent also doesn't directly monitor datastore processes. To see these SQL parameters, you must enable the collection feature in the [agent configuration](/docs/apm/agents/net-agent/configuration/net-agent-configuration/).
```
